generators/wp/templates/wordpress-alpine/docker-compose.yml

```diff
@@ -2,45 +2,46 @@ version: "3.7"
 
 services:
 
+  # php-fpm service & wordpress application code
   wp:
     build:
       context: https://github.com/penske-media-corp/alpine-wordpress.git
       target: wp
       args:
-        NGINX_VERSION: <%= "$NGINX_VERSION" %>
-        NODE_VERSION: <%= "$NODE_VERSION" %>
-        PHP_VERSION: <%= "$PHP_VERSION" %>
-        WEB_ROOT: <%= "$WEB_ROOT" %>
-        WORDPRESS_VERSION: <%= "$WORDPRESS_VERSION" %>
+        NGINX_VERSION: "<%= NGINX_VERSION %>"
+        NODE_VERSION: "<%= NODE_VERSION %>"
+        PHP_VERSION: "<%= PHP_VERSION%>"
+        WEB_ROOT: "<%= WEB_ROOT %>"
+        WORDPRESS_VERSION: "<%= WORDPRESS_VERSION %>"
     depends_on:
       - db
-    env_file: .env
+    env_file: docker-compose.env
     networks:
       - back
       - front
     restart: always
     volumes:
-      - <%= "$WORDPRESS_CACHE:$WEB_ROOT:rw" %>
-      - <%= "$PWD:$WEB_ROOT/wp-content/themes/$WORDPRESS_THEME:rw" %>
-      - <%= "$COMPOSER_VENDOR_DIR/pmc/mu-plugins:$WEB_ROOT/wp-content/mu-plugins:rw" %>
-      - <%= "$COMPOSER_VENDOR_DIR/pmc/plugins:$WEB_ROOT/wp-content/plugins:rw" %>
-      - <%= "$COMPOSER_VENDOR_DIR/pmc/themes/$WORDPRESS_PARENT_THEME:$WEB_ROOT/wp-content/themes/$WORDPRESS_PARENT_THEME:rw" %>
-    working_dir: <%= "$WEB_ROOT" %>
+      - "<%= WORDPRESS_CACHE %>:<%= WEB_ROOT %>:rw"
+      - "<%= WORDPRESS_CACHE %>/wp-content:<%= WEB_ROOT %>/wp-content:rw"
```

Comment:
@hauvong I'm not sure how this might behave for data we want to persist like uploads. I guess we could maybe remap the subdir?

Reply:
This map will persist all uploads under /wp-content, or anything in it.  
.cache/wp <-> /var/www/html
.cache/wp/wp-content <-> /var/www/html/wp-content

So any sub folders under /var/www/wp-content/ will be persist along with any packages we add or install like plugins and themes.

There is one more folder I need to add is 
.cache <-> /var/www
This will make the /var/www persist as there is a few key file that use /var/www folder to hold some .version info to provision the wp core files.  If we update the docker image, we use that info to detect if some package is out of sync and force a reprovision.   This method is add to minimize having to use docker compose to trigger a pull request every time we start up the container.   This just give us a way to force an update only once on any local dev if they pull a new docker wpdev image.